packages/admin/src/styles/admin-m3-overrides.css

```
@@ -304,11 +305,58 @@ body:has(.admin-m3) [data-component="ConfirmDialog"][data-slot="surface"] {
   }
 }
 
+/* ============================================================================
+ * AdminSideSheet motion — the three global AppBar surfaces (Profile,
+ * Settings, Notifications). Mobile keeps the dialog's bottom-sheet slide
+ * (same keyframes, same composition with the Tailwind `translate` centering);
+ * ≥640px slides in from the right edge as an M3 modal side sheet.
+ * ============================================================================ */
+
+@keyframes adminSideSheetIn {
+  from { transform: translateX(100%); opacity: 0.85; }
```

Stylelint (17.14.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)